GSA/digitalgov.gov
# Guide Redesign

## Summary

Redesign of the guides template featuring new header and nav components. Includes:

- Created `guide-header.html` partial to encapsulate the new guide hero
- Created `guide-header-image.html` partial to handle guide hero image logic
- Created `guide-menu-bar.html` partial to display new navigation bar across top of page
- Created `guide-content.html` partial to encapsulate guide content
- Created `attribution.html` partial for attributing header images
- Replaced  `baseof.html` with `single.html` to minimize redundant code
- Added new guide header images under `data/images`
- Added missing guide summaries under `content/guides`
- Created/modified `data/guidenav` files for each guide 

### Preview

| guide             | preview                                                                                                                                                                  | live                                        |
| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------- |
| dap               | [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-new-guides-template/guides/dap/)                   | [live](https://digital.gov/guides/dap/) |
| mobile principles | [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-new-guides-template/guides/mobile-principles/)                   | [live](https://digital.gov/guides/mobile-principles/) |
| public participation               | [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-new-guides-template/guides/public-participation/)                   | [live](https://digital.gov/guides/public-participation/) |
| rpa | [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-new-guides-template/guides/rpa/)                   | [live](https://digital.gov/guides/rpa/) |
| site-scanning               | [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-new-guides-template/guides/site-scanning/)                   | [live](https://digital.gov/guides/site-scanning/) |
| web-analytics-playbook | [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-new-guides-template/guides/web-analytics-playbook/)                   | [live](https://digital.gov/guides/web-analytics-playbook/) |


### Solution

#### Before
<img width="1052" alt="Screenshot 2023-07-12 at 7 05 09 PM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/bc3b9efe-e925-4ca1-a0e0-28fb5a44bdd9">

#### After
<img width="1440" alt="Screenshot 2023-07-31 at 4 08 33 PM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/ae8077fd-4341-459a-8fc4-268db275c10c">


### How To Test
1. Visit each of the guide preview pages linked above
2. Verify that all of the guide content works well in the new template 

- [x] Tested on Google Chrome Desktop (Mac)
- [x] Tested on Google Chrome Desktop (Windows)
- [x] Tested on Google Chrome Mobile (Android)
- [x] Tested on Google Chrome Mobile (iOS)
- [x] Tested on Safari Mobile
- [x] Tested on Firefox Mobile

---

### Dev Checklist
